scripts/bump-builtin-connectors.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Bumps the versions of built-in connectors
+
+git checkout main
+git pull origin main
+git checkout -b bump-builtin-connectors
+
+go get github.com/conduitio/conduit-connector-file@latest
```

Comment:
I liked the second script with the `conn` loop, I think it would make this script smaller and nicer, wdyt?

Reply:
You're right, I have no idea why I went with this.:O